src/styles/cards.scss

```diff
@@ -0,0 +1,101 @@
+@import "./stars";
+
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
No need to hardcode height for the card, it will be better if the height depends on the content